### PR TITLE
Add home screen

### PR DESCRIPTION
**Pull request by mellowgeek**
First cut at removing bottom navigation and adding the home screen.

### CODE REVIEW

File: app/src/main/java/org/coepi/android/ui/home/HomeFragment.kt [hunk 1]
```diff
@@ -0,0 +1,43 @@
+package org.coepi.android.ui.home
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.navigation.fragment.findNavController
+import org.coepi.android.databinding.FragmentHomeBinding.inflate
+import kotlinx.android.synthetic.main.fragment_home.*
```
Comment (i-schuetz) [inline app/src/main/java/org/coepi/android/ui/home/HomeFragment.kt]: Don't use kotlinx. We use View Binding. See other fragments.

File: app/src/main/java/org/coepi/android/ui/home/HomeFragment.kt [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    ): View? = inflate(inflater, container, false).apply {
+        lifecycleOwner = viewLifecycleOwner
+        vm = viewModel
+    }.root
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+
+        check_in.setOnClickListener {
```
Comment (i-schuetz) [inline app/src/main/java/org/coepi/android/ui/home/HomeFragment.kt]: Put this in `onCreateView`, see other fragments

File: app/src/main/java/org/coepi/android/ui/home/HomeFragment.kt [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+            val action = HomeFragmentDirections.actionHomeFragmentToCareFragment()
+
+            findNavController().navigate(action)
```
Comment (i-schuetz) [inline app/src/main/java/org/coepi/android/ui/home/HomeFragment.kt]: Navigation should be in view model, see existing view models

File: app/src/main/res/layout/fragment_home.xml [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+            android:layout_width="344dp"
+            android:layout_height="125dp"
+            app:cardBackgroundColor="#FAFAFA"
+            app:cardCornerRadius="8dp"
+            android:layout_marginTop="24dp"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
+            app:layout_constraintEnd_toEndOf="parent">
+
+            <RelativeLayout
```
Comment (i-schuetz) [inline app/src/main/res/layout/fragment_home.xml]: Use `ConstraintLayout`. It pretty much replaces `RelativeLayout`.

File: app/src/main/res/layout/fragment_home.xml [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+    xmlns:tools="http://schemas.android.com/tools">
+
+    <data>
+
+        <variable
+            name="vm"
+            type="org.coepi.android.ui.home.HomeViewModel" />
+    </data>
+
+    <androidx.constraintlayout.widget.ConstraintLayout
```
Comment (i-schuetz) [inline app/src/main/res/layout/fragment_home.xml]: You can use a vertical `LinearLayout`.

File: app/src/main/java/org/coepi/android/ui/container/TabsContainerFragment.kt [hunk 6]
```diff
@@ -5,13 +5,7 @@ import android.view.LayoutInflater
...
-import androidx.navigation.Navigation
-import androidx.navigation.ui.setupWithNavController
-import com.google.android.material.bottomnavigation.BottomNavigationView
-import org.coepi.android.R.id.bottom_navigation
-import org.coepi.android.R.id.navHostFragment
 import org.coepi.android.databinding.FragmentTabsContainerBinding.inflate
-import org.coepi.android.ui.onboarding.OnboardingViewModel
 import org.koin.androidx.viewmodel.ext.android.viewModel
 
 class TabsContainerFragment : Fragment() {
```
Comment (i-schuetz) [inline app/src/main/java/org/coepi/android/ui/container/TabsContainerFragment.kt]: Rename it in `ContainerFragment`. Probably the class and navigation root should be removed, but let's keep it for now.

**Review by ivnsch — CHANGES_REQUESTED** [review]
Since there are no tabs anymore, can you add an entry point to the BLE fragment in the debug view?

**Comment by mellowgeek** [comment]
> Since there are no tabs anymore, can you add an entry point to the BLE fragment in the debug view?

Added a BLE button to the debug log screen.

**Review by ivnsch — APPROVED** [review]

**Pull request merged**